src/core/jpx.js

```diff
@@ -192,7 +192,6 @@ class JpxImage {
                 XRsiz: data[j + 1],
                 YRsiz: data[j + 2],
               };
-              j += 3;
```

Comment:
Sorry, but this is definitely *not* correct since it'll incorrectly use the same data for each component; please revert this.

Reply:
yeah，I see